README.md

```diff
@@ -265,6 +265,154 @@ end)
 
 10. Install the dependencies below.
 
+11. (OPTIONAL) If you want to use [qb-doorlock](https://github.com/qbcore-framework/qb-doorlock) to lock doors inside your shell then add the following code into `qb-doorlock > client > main.lua > around line 812` under `RegisterKeyMapping('remotetriggerdoor', Lang:t("general.keymapping_remotetriggerdoor"), 'keyboard', 'H')`
+```lua
+function generateRandomString(length)
```

Comment:
This can be simplified a bit
https://stackoverflow.com/questions/71541205/how-do-i-make-a-string-in-lua-contain-random-characters
Ex: 
local randuppercase = string.char(math.random(65, 65 + 25))
local randlowercase = string.char(math.random(65, 65 + 25)):lower()